### ISSUE TITLE
Freezing Bug When I Sleep or Lid My laptop

### ISSUE THREAD

**Issue by binocry** [brave/brave-browser#14582]
## Description 

When I open Brave with 1 tab or more, if I try to sleep my laptop or lid it, Brave will always freezing, really doesn't respond to anything but strangely the minimize and close buttons function normally. I tried to do the same on Microsoft Edge where is Chromium browser fork it's works smoothly without any problems. please fix this bug.

## Steps to Reproduce

   1. Open Brave Browser
   2. Open 1 or more tab
   3. Try to sleep or lid my laptop

## Actual result:

should be able to resume as usual

## Expected result:
Brave Freezing and all I can do is minimize and close

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Versi 1.21.74 Chromium: 89.0.4389.72 (Build Resmi) (64 bit)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Only trying in release channel
- Can you reproduce this issue with the beta channel? dont know
- Can you reproduce this issue with the nightly channel? dont know

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

DESKTOP-NTGKELH Processor AMD A8-7410 APU with AMD Radeon R5 Graphics 2.20 GHz Installed RAM 4,00 GB (3,41 GB usable) Device ID 7439C0ED-AE08-45DC-9B9D-0D2C78C54FEA Product ID 00326-10000-00000-AA258 System type 64-bit operating system, x64-based processor Pen and touch No pen or touch input is available for this display

**Comment by binocry** [comment]
so any progress? i still had problem in latest version 1.22.71 Chromium: 89.0.4389.114 (Build Resmi) (64 bit)

**Comment by rebron** [comment]
@binocry Latest version is 1.24.85 Chromium: 90.0.4430.212. Can you update to that and let us know if you're still having freezing issues. Also, can you check the brave://settings/system the hardware acceleration setting to see if that maybe causing the freeze issue.

**Comment by binocry** [comment]
> @binocry Latest version is 1.24.85 Chromium: 90.0.4430.212. Can you update to that and let us know if you're still having freezing issues. Also, can you check the brave://settings/system the hardware acceleration setting to see if that maybe causing the freeze issue.

I'm always updated. The freezing is very random, sometimes it can run normally but usually it always freezes after waking from sleep

ok I tried turning off the hardware accelerator. the first results were promising

**Comment by revanmj** [comment]
I have the same issue on version 1.48.171 on my laptop (Lenovo Legion 5) with Windows 11 Pro. 

Almost every time laptop goes to sleep (usually after set time of inactivity), Brave UI freezes after waking up (window reacts to clicking minimize/maximize buttons, cursor changes when going over address bar, but Brave UI itself does not respond (buttons do not hightlight on hover, etc.).

**Comment by binocry** [comment]
> I have the same issue on version 1.48.171 on my laptop (Lenovo Legion 5) with Windows 11 Pro.
> 
> Almost every time laptop goes to sleep (usually after set time of inactivity), Brave UI freezes after waking up (window reacts to clicking minimize/maximize buttons, cursor changes when going over address bar, but Brave UI itself does not respond (buttons do not hightlight on hover, etc.).
> 
> Also confirm that MS Edge does not have this problem (I usually have both Brave and Edge opened).

yeah the only solution is disable hardware acceleration on settings, but it make browsing so slow and my cpu so hot
